techniques/system/common/1.0/promises.st

```diff
@@ -62,43 +91,47 @@ bundle common va
         "check_red_button_status",
         "check_cf_processes",
         "check_uuid",
-        "configure_rudder_reporting_system",
         "garbage_collection",
         "setup_cronjob",
         "check_cron_daemon",
-        "check_binaries_freshness",
+&if(INITIAL)&
+        "fetchFusionTools",
+        "virtualMachines",
+        "doInventory",
+&else&
+        "configure_rudder_reporting_system",
         "check_zypper",
         "get_environment_variables",
         "properties",
         "nxlog_enable",
-        "configuration"
+&endif&
       };
 
-    !policy_server::
+    policy_server::
 
-      "bs" slist => {
-        "startup",
-        "check_disable_agent",
-        "clean_red_button",
-        "update",
-        "configuration",
-        "set_red_button",
-        "internal_security",
-        "check_red_button_status",
-        "check_cf_processes",
-        "garbage_collection",
-        "setup_cronjob",
-        "check_cron_daemon",
+      "specific_bs" slist => {
+&if(INITIAL)&
+        "install_rsyslogd",
+        "propagatePromises",
+        "sendInventoryToCmdb",
+        "root_component_check",
+&endif&
+        # This bundle is duplicated to avoid having empty lists and cfnull in bundlesequence
         "check_binaries_freshness",
-        "configure_rudder_reporting_system",
-        "nxlog_enable",
-        "check_zypper",
-        "check_uuid",
-        "get_environment_variables",
-        "properties",
-        "configuration"
       };
 
+&if(INITIAL)&
+      # initial promises server specific inputs cannot be loaded from rudder_system_directives.cf
```

Comment:
Because they are generated there in a String Template variable that is dynamic in the webapp but static in initial promises. So we need a condition to choose them at runtime with a class here and not at generation time.